vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,94 @@
+def call(Map pipelineParams) {
+    pipeline {
+        agent any
+        parameters {
+            string(name: 'scylla_version', defaultValue: '', description: 'Scylla version to test')
+            string(name: 'base_versions', defaultValue: '', description: 'Base versions')
+            string(name: 'provision_type', defaultValue: '', description: 'Provision type')
+            string(name: 'new_scylla_repo', defaultValue: '', description: 'New Scylla repo')
+        }
+        stages {
+            stage('Get Scylla Version') {
+                steps {
+                    script {
+                        def matrix_version = params.scylla_version?.trim()
+                        def job = "release-triggers/performance/perf-regression-trigger"  // not created yet
+                        if (matrix_version == "master:latest") {
+                            matrix_version = "master"
```

Comment:
let add a pipeline parameter `matrix_version` that can be overridden by the user

like `matrix_version=master-vnodes` and such, so we can build specific unique groups for scheduling needs

while the official release can pass along scylla_version that we can use.

Reply:
@fruch do you mean to use this parameter `matrix_version` for master only? I do not understand the parameter name

Reply:
for anything that we schedule, we need to know what it is, just using `scylla_version=master:latest` as a way to understand the difference isn't enough